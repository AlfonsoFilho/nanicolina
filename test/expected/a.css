.a {
  padding-right: 15px;
  padding-left: 15px;
  margin-right: auto;
  margin-left: auto;
}
.b {
  margin-right: -15px;
  margin-left: -15px;
}
.c,
.d,
.e,
.f,
.g,
.h,
.i,
.j,
.k,
.l,
.m,
.n,
.o,
.p,
.q,
.r,
.s,
.t,
.u,
.v,
.w,
.x,
.y,
.z,
.ba,
.bb,
.bc,
.bd,
.be,
.bf,
.bg,
.bh,
.bi,
.bj,
.bk,
.bl,
.bm,
.bn,
.bo,
.bp,
.bq,
.br,
.bs,
.bt,
.bu,
.bv,
.bw,
.bx {
  position: relative;
  min-height: 1px;
  padding-right: 15px;
  padding-left: 15px;
}
@media all and (max-width: 300px){
  .b {
    margin-right: 0;
  }
}
