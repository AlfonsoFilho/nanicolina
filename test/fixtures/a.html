<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Test A</title>
</head>
<body>
  <div class="container-fluid">
    <div class="row">
      <div class="col-xs-1 col-sm-8"></div>
      <div class="col-xs-1 col-sm-8"></div>
    </div>
    <div class="row hidden">
      <div class="col-xs-12 col-lg-6"></div>
    </div>
  </div>
</body>
</html>
