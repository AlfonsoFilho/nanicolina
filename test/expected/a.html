<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Test A</title>
</head>
<body>
  <div class="a" ng-class="{'Z bb': show()}">
    <div class="b">
      <div class="c F"></div>
      <div class="c F"></div>
    </div>
    <div class="b bb">
      <div class="U z"></div>
    </div>
  </div>
</body>
</html>
